### PR TITLE
Fix oc link

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: createdisk.sh [hunk 1]
```diff
@@ -72,7 +72,7 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl unmask chronyd
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl disable chronyd
 
 # Enable the podman.socket service for API V2
-    ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl enable podman.socket
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo systemctl enable podman.socket
```
Comment (gbraad) [inline createdisk.sh:75]: unrelated ?

**Review by gbraad — CHANGES_REQUESTED** [review]
unnecessary change to podman socket

File: createdisk.sh (hunk 1 above)
Comment (praveenkumar) [reply createdisk.sh:75]: @gbraad this is part of different commit.

**Review by guillaumerose — APPROVED** [review]

File: snc.sh [hunk 2]
```diff
@@ -45,16 +45,9 @@ fi
...
-    curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-mac-${OPENSHIFT_RELEASE_VERSION}.tar.gz" | tar -zx -C openshift-clients/mac oc
-    curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-windows-${OPENSHIFT_RELEASE_VERSION}.zip" > openshift-clients/windows/oc.zip
-else
-    curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-linux.tar.gz" | tar -zx -C openshift-clients/linux oc
-    curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-mac.tar.gz" | tar -zx -C openshift-clients/mac oc
-    curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-windows.zip" > openshift-clients/windows/oc.zip
-fi
+curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-linux.tar.gz" | tar -zx -C openshift-clients/linux oc
+curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-mac.tar.gz" | tar -zx -C openshift-clients/mac oc
+curl -L "${MIRROR}/${OPENSHIFT_RELEASE_VERSION}/openshift-client-windows.zip" > openshift-clients/windows/oc.zip
```
Comment (cfergeau) [inline snc.sh:50]: For OKD releases, `MIRROR` will point at (for example) https://github.com/openshift/okd/releases/tag/4.6.0-0.okd-2020-12-12-135354 which does not have unversioned tarballs. If you want to use the same URL for both, you should be using the versioned ones.

Comment (praveenkumar) [reply snc.sh:50]: Good catch, I misinterpreted the `OKD_VERSION` logic :(

**Review by cfergeau — APPROVED** [review]

**Review by gbraad — APPROVED** [review]
